tests/Xamarin.Android.Tools.AndroidSdk-Tests/AndroidSdkWindowsTests.cs

```diff
@@ -13,56 +13,5 @@ namespace Xamarin.Android.Tools.Tests
 	[TestFixture]
 	public class AndroidSdkWindowsTests
 	{
-		[Test]
-		public void ExtractVersion ()
```

Comment:
Did this test get replaced by a new test?

Reply:
No; the method it was testing was removed, as the "let's extract a version code from a filesystem path" code was "replaced" with the (far older) "parse the version from the `release` file".